decidim-verifications/spec/events/decidim/verifications/managed_user_error_event_spec.rb

```diff
@@ -7,7 +7,7 @@
 
   let(:event_name) { "decidim.events.verifications.managed_user_error_event" }
   let(:resource) { create :conflict }
-  let(:organization_host) { resource.current_user.organization.host }
+  let(:organization_host) { "#{resource.current_user.organization.host}" }
```

Comment:
There is a linting error with this line.

I think you can just revert it back to original.

Reply:
Done at 88d2d8a63b66467870cb3f75ebb9617959b26858